pegasus/sites.v3/code.org/views/how_ai_works_resources.haml

```diff
@@ -3,15 +3,16 @@
     .content-wrapper
       %p.overline
         =hoc_s(:teachers_label)
-      %h3=hoc_s(:ai_pl_101_hero_heading)
-      %img{src: "/shared/images/announcement/announcement_ai_pl_2023.png", alt: ""}
-      %p=hoc_s(:ai_pl_101_hero_desc)
+      %h3=hoc_s(:ai_pl_101_title)
+      %img{src: "/images/self-paced-pl-tile-ai-101.jpg", alt: ""}
+      %p=hoc_s(:ai_pl_101_desc)
     .content-footer
       %p
-        %strong=hoc_s(:module_label_duration)
+        %strong
+          =hoc_s(:module_label_duration)
         =hoc_s(:module_duration_5_hrs)
-      %a.link-button{href: "/ai/pl/101", aria:{label: hoc_s(:aria_label_call_to_action_ai_pl_101_videos)}}
-        =hoc_s(:call_to_action_watch_videos)
+      %a.link-button{href: CDO.studio_url("/courses/self-paced-pl-ai-101")}
```

Comment:
Is it okay that the aria label is being removed?

Reply:
Just added one! It'll say `Start the AI 101 self-paced professional learning program` since we already have that string.